src/components/cards/EncountersCard.jsx

```diff
@@ -58,31 +58,56 @@ export default function EncountersCard({
       reference: 'date',
       name: 'time',
       label: 'Date',
-      options: { cellRenderer: cellRendererTypes.specifiedTime },
+      options: {
+        cellRenderer: cellRendererTypes.specifiedTime,
+        cellRendererProps: { noWrap: true },
+        width: '20%',
+      },
     },
     {
       reference: 'location',
       name: 'formattedLocation',
       label: 'Location',
-      options: { cellRenderer: cellRendererTypes.location },
+      options: {
+        cellRenderer: cellRendererTypes.location,
+        cellRendererProps: { noWrap: true },
+      },
     },
     {
       reference: 'owner',
       name: 'owner',
       label: 'Owner',
-      options: { cellRenderer: cellRendererTypes.user },
+      options: {
+        cellRenderer: cellRendererTypes.user,
+        cellRendererProps: { noWrap: true },
+      },
     },
     {
       reference: 'actions',
       name: 'guid',
       label: 'Actions',
       options: {
+        width: (function buildWidth() {
```

Comment:
this is cool, feel like it could benefit from abstraction though since we would probably want this set on all action columns in all tables. a few ideas: 
 - build an actions cellRenderer and accept a `numberOfActions` prop for this renderer
 - create a hook that returns the `buildActionColumnWidth` function 
 - something else?

Reply:
Created a hook in https://github.com/WildMeOrg/codex-frontend/pull/314/commits/9360ac70598a7a71a19de6dd510d94321b772ecc